src/cloud/docker/docker-quick-reference.md

```diff
@@ -46,15 +48,68 @@ docker-compose -f docker-compose.yml -f docker-compose-custom.yml [-f more-custo
 
 ### Additional build options
 
-| Option       | Key              | Available values
-| ------------ | ---------------- | ------------------
-| [Mode][]         | `--mode`, `-m`   | production, developer
-| [File synchronization engine][] | `--sync-engine` | native (default), docker-sync, mutagen
-| Specify a custom URL for Magento | `--host`<br>`--port`
-| Specify custom HTTP and SMTP ports for MailHog | `--mailhog-http-port`<br>`--mailhog-smtp-port`
+View the available options for the `ece-docker build:compose` command:
+
+```bash
+php ./vendor/bin/ece-docker build:compose -h
+```
+
+```terminal
```

Comment:
My plan is to automate the output for all of the CLI commands. I put this here because the service-versions table is hard to maintain, and there are many options that are not included there.